src/EnergyPlus/DataHeatBalance.cc

```diff
@@ -491,13 +491,15 @@ namespace DataHeatBalance {
     int OverallHeatTransferSolutionAlgo(DataSurfaces::HeatTransferModel_CTF); // Global HeatBalanceAlgorithm setting
 
     // Flags for HeatTransfer Algorithms Used
+    bool AllCTF(true);                      // CTF used for everything - no EMPD, no CondFD, No HAMT, No Kiva - true until flipped otherwise
```

Comment:
This flag is used to invoke the new `CalcHeatBalanceInsideSurf2CTFOnly` function.

Reply:
Having some extra precalculated flags makes sense, allowing us to make better decisions about what functions to call later on.